warehouse/query-core/src/main/java/datawave/query/iterator/logic/AndIterator.java

```diff
@@ -367,12 +384,13 @@ public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean i
                     // throw IterationInterrupted exceptions as-is with no modifications so the QueryIterator can handle it
                     throw e2;
                 } catch (WaitWindowOverrunException e) {
+                    log.warn(id + ": Passing through WaitWindowOverrunException: " + e.getMessage());
```

Comment:
```suggestion
                    log.warn(id + ": AndIterator.seek() passing through WaitWindowOverrunException: " + e.getMessage());
```

Reply:
I think only one of the "passing through" logs is necessary here in the nested try/catch blocks

Reply:
Should these "passing through" logs really be at WARN?  Also, why just in AndIterator and not OrIterator.